pkg/tsdb/influxdb/influxql/converter/testdata/influx_select_*_from_cpu-frame.jsonc

```diff
@@ -0,0 +1,166 @@
+//  🌟 This was machine generated.  Do not edit. 🌟
+//  
+//  Frame[0] 
+//  Name: cpu.mean_usage_guest
```

Comment:
when looking at the input vs output, i would expect the parser to produce a frame with name "cpu" and fields matching the names in:
```
          "name": "cpu",
          "columns": [
            "time",
            "mean_usage_guest",
            "mean_usage_nice",
            "mean_usage_idle"
          ],
```
if we then need to split that in to [timeseries multi](https://grafana.github.io/dataplane/timeseries#time-series-multi-format-timeseriesmulti), that is a straightforward task (but should not be done as part of parsing)

Reply:
I can produce the result that way. Is that what we want to achieve in general?

Reply:
In general i think there should be the fewest *changes* to from the input format to the output.  The input values here are structured as a table, so keep them as a table :)

Reply:
This is not a table. This is a general response from InfluxDB. See here https://docs.influxdata.com/influxdb/v2/api/#operation/GetLegacyQuery 
To able to create timeseries data we were converting it to data frames already. 

On Frontend Mode
- https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/influxdb/datasource.ts#L629-L634
- https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/influxdb/influx_series.ts#L21-L67

On Backend Mode
- https://github.com/grafana/grafana/blob/main/pkg/tsdb/influxdb/influxql/response_parser.go#L77-L115